docs/hub/models-gated.md

```diff
@@ -35,60 +78,48 @@ extra_gated_fields:
-<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/models-gated-manual-approval.png"/>
-<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/models-gated-manual-approval-dark.png"/>
-</div>
+## Access gated models as a user
 
-You can automate the approval of access requests with the following API:
+### Request access
 
-| Method | URI | Description | Payload |
-| ------ | --- | ----------- | -------  |
-| `GET` | `/api/models/{repo_id}/user-access-request/pending` | Retrieve the list of pending access requests for the given model. | ```headers = { "authorization" :  "Bearer $token" }``` |
-| `GET` | `/api/models/{repo_id}/user-access-request/accepted` | Retrieve the list of accepted access requests for the given model. | ```headers = { "authorization" :  "Bearer $token" }``` |
-| `GET` | `/api/models/{repo_id}/user-access-request/rejected` | Retrieve the list of rejected access requests for the given model | ```headers = { "authorization" :  "Bearer $token" }``` |
-| `POST` | `/api/models/{repo_id}/user-access-request/handle` | Change the status of a given access request to `status`. | ```headers = { "authorization" :  "Bearer $token" }``` ```json = { "status": "accepted" &#124; "rejected" &#124; "pending", "user": "username" }``` |
-| `POST` | `/api/models/{repo_id}/user-access-request/grant` | Allow a specific user to access your repo. | ```headers = { "authorization" :  "Bearer $token" }``` ```json = { "user": "username" } ``` |
+As a user, if you want to use a gated model, you will need to request access to it. This mean that only registered users can access gated models.
 
+Requesting access can only be done from your browser. Go to the model on the Hub and you will be prompted to share your information:
 
-The base URL for the HTTP endpoints above is `https://huggingface.co`. The `$token` to pass as a bearer token can be generated from [your user settings](https://huggingface.co/settings/tokens). It must have `write` access to the gated repository.
+![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/models-gated-user-side.png)
 
-### Notifications settings
+By clicking on `Agree`, you agree to share your username and email address to the model authors. In some cases, additional fields might be requested. Try to fill the form as accurately as possible to help the model authors decide on whether they grant you the access or not.
 
-By default, notifications for new pending access requests are sent once a day via email.
-When the repo lives in an organization, those emails are sent to the first 5 admins of the organization.
+Once the access request is sent, there are two possibilities. If the approval mechanism is automatic and you immediately get access to the model files. Otherwise, the requests have to be approved manually by the authors which can take more time. 
```

Comment:
```suggestion
Once the access request is sent, there are two possibilities. If the approval mechanism is automatic, you immediately get access to the model files. Otherwise, the requests have to be approved manually by the authors, which can take more time. 
```